app/frontend/src/components/Calendar/components/Cells.jsx

```diff
@@ -77,7 +77,7 @@ const dateLabelStatus = {
   yes: theme.white,
 };
 
-const Cells = ({currentDate, onDateClick, userIdList}) => {
+const Cells = React.memo(({currentDate, onDateClick, userIdList}) => {
```

Comment:
why not useMemo?

Reply:
because this is different kind of memozation.
I wanted to memo the whole componenet (which is the equivilant to PureComponent)
the way react suggests doing so is with react.memo
https://reactjs.org/docs/hooks-faq.html#how-do-i-implement-shouldcomponentupdate